pom.xml

```diff
@@ -184,6 +184,11 @@
                 <artifactId>mockito-core</artifactId>
                 <version>${mockito.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-inline</artifactId>
+                <version>${mockito.version}</version>
+            </dependency>
```

Comment:
Could we just update mockito to 5.4.0 to get rid of inline as that would have it baked in already?

Reply:
Good point! I'll bump the version